Add kaamiki setup and update README.md

stardust-xs opened the pull request:
_Oct 11, 2020 - v0.0.1_

**Added:**
- Author name in license block in **README.md**.
- Necessary files for setting up (installing) @kaamiki on other machines.

stardust-xs committed:
Merge pull request #1 from kaamiki/master

Add author name in license block

Commit:
Upstream files to git
Oct 11, 2020 - v0.0.1

Added:
- Kaamiki setup files.

Commit:
Update kaamiki setup

Commit:
Update README.md

Commit:
Add and update Kaamiki's Code of Conduct
Oct 11, 2020 - v0.0.1

Added:
- Content of Kaamiki's Code of Conduct based on @kwattorama's CoC.
- Code of Conduct banner.
- Email address for reporting violations.

xames3 commented:
Please review [this](https://github.com/kaamiki/kaamiki/pull/2/commits/c14a71651c43a632799ac659af5a2c3258a01018) commit @PranaliRPatil , @kwattorama and @srushti-maladkar.

srushti-maladkar commented on setup.py:
```diff
@@ -0,0 +1,127 @@
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Home Automation",
+        "Topic:: Security",
+        "Topic:: Security:: Cryptography",
```
Spaces mismatch
Expected "Topic :: Security" to maintain uniform format

stardust-xs replied:
Fixed [here](https://github.com/kaamiki/kaamiki/commit/0ca87ca7a5df97ed3f246591da8c2ae0d63a2afe), please check.

srushti-maladkar reviewed (changes requested):
Comments to be resolved before merge

PranaliRPatil reviewed (approved):
Looks fine.

Commit:
Fix space mismatch in pypi classifiers
Oct 12, 2020 - v0.0.1

Fixed:
- Space mismatch in classifiers.
  Here: https://github.com/kaamiki/kaamiki/pull/2#discussion_r503135816

srushti-maladkar reviewed (approved):
Approved

kwattorama commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,77 @@
+
+- We welcome and encourage participation by everyone and are committed to being in a community where everyone feels good about joining.
+- We sacredly believe in the concept of **[Vasudhaiv Kutumbkam](https://en.wikipedia.org/wiki/Vasudhaiva_Kutumbakam)** (The World is One Family) in letter and spirit. Although we may not be able to satisfy everyone, we will always work to treat everyone well.
+- We don't encourage support and suggestions on demand, we expect everyone to be proactive. We believe in proactive help or contribution to a problem. While doing so, please be careful not to be too patronizing or disrespectful. When someone turns to you for help, always be open, listen and try to put yourself in the position of your counterpart. If you don't have time, let the person know or may ...
```
this point will relate more in the expected behavior section

stardust-xs replied:
Rearranged [here](https://github.com/kaamiki/kaamiki/commit/70d1aad8ee5bcf65a38d389ef53fb58e4c9508fc).

kwattorama commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,77 @@
+  
+## Reporting Violations
+
+- If you are the subject of, or witness to any violations of this Code of Conduct, please email us at <xames3.kaamiki@gmail.com> with an incident report (if possible).
```
Hyperlink "incident report" with incident reporting google form.

stardust-xs replied:
Added [here](https://github.com/kaamiki/kaamiki/commit/70d1aad8ee5bcf65a38d389ef53fb58e4c9508fc), please check.

kwattorama committed:
Create CODE_OF_CONDUCT_HN.md

add CoC in Hindi language

Commit:
Update CODE_OF_CONDUCT_HN with proper links
Oct 14, 2020 - v1.0.1

Added:
- Incident report form link.

Changed:
- Refernce hyperlink to Vasudhaiva Kutumbakam
- Order of points
- Refactor for spaces.

Commit:
Update CODE_OF_CONDUCT with proper links
Oct 14, 2020 - v1.0.1

Added:
- Incident report form link.

Changed:
- Reference hyperlink to Vasudhaiva Kutumbakam
- Order of points

kwattorama reviewed (approved):
Perfect!
